aacgmv2/wrapper.py

```diff
@@ -200,91 +239,78 @@ def convert_latlon_arr(in_lat, in_lon, height, dtime, code="G2A"):
     At least one of in_lat, in_lon, and height must be a list or array.
     """
     import aacgmv2._aacgmv2 as c_aacgmv2
+    import aacgmv2
 
-    # If a list was entered instead of a numpy array, recast it here
-    if isinstance(in_lat, list):
-        in_lat = np.array(in_lat)
-
-    if isinstance(in_lon, list):
-        in_lon = np.array(in_lon)
-
-    if isinstance(height, list):
-        height = np.array(height)
+    # Recast the data as numpy arrays
+    in_lat = np.array(in_lat)
+    in_lon = np.array(in_lon)
+    height = np.array(height)
 
     # If one or two of these elements is a float or int, create an array
-    test_array = np.array([hasattr(in_lat, "shape"), hasattr(in_lon, "shape"),
-                           hasattr(height, "shape")])
-    if not test_array.all():
-        if test_array.any():
-            arr_shape = in_lat.shape if test_array.argmax() == 0 else \
-                        (in_lon.shape if test_array.argmax() == 1 else
-                         height.shape)
-            if not test_array[0]:
-                in_lat = np.ones(shape=arr_shape, dtype=float) * in_lat
-            if not test_array[1]:
-                in_lon = np.ones(shape=arr_shape, dtype=float) * in_lon
-            if not test_array[2]:
-                height = np.ones(shape=arr_shape, dtype=float) * height
-        else:
-            logging.info("for a single location, consider using convert_latlon")
+    test_array = np.array([len(in_lat.shape), len(in_lon.shape),
+                           len(height.shape)])
+    if test_array.min() == 0:
+        if test_array.max() == 0:
+            aacgmv2.logger.warning("for a single location, consider using " \
+                                   "convert_latlon or get_aacgm_coord")
             in_lat = np.array([in_lat])
             in_lon = np.array([in_lon])
             height = np.array([height])
+        else:
+            imax = test_array.argmax()
+            max_shape = in_lat.shape if imax == 0 else (in_lon.shape \
+                                            if imax == 1 else height.shape) 
+            if test_array[0] == 0:
+                in_lat = np.full(shape=max_shape, fill_value=in_lat)
+            if not test_array[1]:
```

Comment:
This allows the user to input only one of the locations as an array and specify the others as a single value that will be the same for all calculations.  I changed the logic in the if statement to be consistent, since they're all doing the same thing.